pom.xml

```diff
@@ -38,20 +38,21 @@
 
         <!-- On this block, library versions must be kept in sync with the ones in the quarkus-bom: -->
         <!-- Start-->
-        <quarkus.version>3.5.3</quarkus.version>
+        <quarkus.version>3.8.3</quarkus.version>
         <netty-bom.version>4.1.107.Final</netty-bom.version>
-        <opentelemetry.version>1.31.0</opentelemetry.version>
-        <opentelemetry-alpha.version>1.31.0-alpha</opentelemetry-alpha.version>
-        <smallrye-config.version>3.5.4</smallrye-config.version>
+        <opentelemetry.version>1.32.0</opentelemetry.version>
```

Comment:
It would be great to use the last OTel version in the Quarkus OTel extension and in the Quarkus OTel exporters ;)

Reply:
Not going to happen soon because of semantic conventions.